grails-app/migrations/0.8.x/changelog-2023-09-21-1400-add-eventtypes-for-request-approval-workflow.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="kchelstowski" id="210920231400-1">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="event_type" />
+        </preConditions>
+        <insert tableName="event_type">
+            <column name="id" value="abb5b3d2-da3e-4a4c-9e3c-de139147f822"/>
+            <column name="version" valueNumeric="0"/>
+            <column name="date_created" valueDate="2023-09-21T00:00:00.0"/>
```

Comment:
Perhaps it's better to use `<property name="now" value="now()"/>` for these dates.

Reply:
Interesting. I hadn't really thought about the use of now() until you mentioned it here. I'm not sure I agree, but I'm fine with either approach. There's something to be said for having a static value for static data to ensure that everyone has the exact same data, down to the timestamps. 

There's also an argument for knowing the exact time the data was created in case you are troubleshooting a database migration issue during deployment (i.e. a failed precondition). 

And just from an aesthetic point of view, sometimes when I do a `select * from event_type` or some other static data table, there's a nice fuzzy feeling of seeing 2012 in the date_created. That is usually followed quickly with dread and despair of knowing I'm old af. 

Ok then to prevent existential crises ... let's use now().

Reply:
Definitely we should avoid existential crises :laughing: so I'll change that